tests/test_opds_import.py

```diff
@@ -89,7 +94,16 @@ def update_work_for_edition(self, edition, *args, **kwargs):
             raise Exception("Utter work failure!")
 
 
-class TestMetadataWranglerOPDSLookup(DatabaseTest):
+class OPDSTest(DatabaseTest):
+    """A unit test that knows how to find OPDS files for use in tests."""
+
+    def sample_opds(self, filename):
```

Comment:
Later on in this file there's a lot of duplicative code for reading sample OPDS files from disk. Bringing that code up here to a superclass lets me simplify the old code and write `test__annotate_feed_response` in a simple way.